spec/models/content_block_spec.rb

```diff
@@ -5,6 +5,20 @@
 RSpec.describe ContentBlock, type: :model do
   it { is_expected.to belong_to(:content_blockable) }
   it { is_expected.to have_many(:media_contents) }
+
+  ["title", "intro", "body"].each do |column|
+    it "should remove emojis from #{column} before saving" do
+      content_block =  FactoryBot.build(:content_block, {
+        intro: "test😊😍😌🤕👿👹👧👧🏻👧🏼👧🏽🏼👍🏽👌☝🏼🥝🥦🌶🌽🍎",
+        title: "test😊😍😌🤕👿👹👧👧🏻👧🏼👧🏽🏼👍🏽👌☝🏼🥝🥦🌶🌽🍎",
+        body: "test😊😍😌🤕👿👹👧👧🏻👧🏼👧🏽🏼👍🏽👌☝🏼🥝🥦🌶🌽🍎",
+        content_blockable: FactoryBot.create(:news_item)
+      })
```

Comment:
wouldn't something like the following work to reduce the triple same stuff?

```
content_block =  FactoryBot.build(:content_block, {
  [column]: "test😊😍😌🤕👿👹👧👧🏻👧🏼👧🏽🏼👍🏽👌☝🏼🥝🥦🌶🌽🍎",
  content_blockable: FactoryBot.create(:news_item)
})
```

but did not work out for me testing it this way

Reply:
Fixed in ff76324